timeusage/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+from . import views
+from .apps import TimeUsageConfig
+
+app_name = TimeUsageConfig.name
+urlpatterns = [
+    url(r'^timeusage/(?P<course_id>\d+)/$',
```

Comment:
This app should be mounted with timeusage in the main url config. See the other comment.